vulnerabilities/cognito-user-group-spoofing.yaml

```diff
@@ -0,0 +1,27 @@
+title: Cognito User Group Spoofing on AWS
```

Comment:
"Cognito user group spoofing"
I think "AWS" is redundant here since we have the affected platform field and Cognito is well known